app/views/matches/index.html.slim

```diff
@@ -23,17 +25,17 @@
           - if match['winner']=='b'
             span.blue-text
               = match['blue_goal']
-            | -
+            | &nbsp-&nbsp
```

Comment:
@weiqingtoh `&nbsp` is not to escape strings but it is a "no break space" to add a whitespace character.